code/__HELPERS/matrices.dm

```diff
@@ -63,3 +63,92 @@
 //The Y pixel offset of this matrix
 /matrix/proc/get_y_shift()
 	. = f
+
+/datum/ColorMatrix
```

Comment:
This doesn't really fit our code standards for titles, please make it camel-case (`color_matrix`).